src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/NotificationClaimantOfHearingHandler.java

```diff
@@ -72,31 +65,67 @@ public List<CaseEvent> handledEvents() {
+                legacyCaseRef,
+                HEARING_FEE,
+                caseData.getHearingFee() == null ? "£0.00" : String.valueOf(caseData.getHearingFee().formData()),
+                HEARING_DATE,
+                hearingDate,
+                HEARING_TIME,
+                hearingTime,
+                HEARING_DUE_DATE,
+                caseData.getHearingDueDate() == null ? "" :
+                    caseData.getHearingDueDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy")),
+                CLAIMANT_REFERENCE_NUMBER, reference
+            ));
         } else {
-            reference = caseData.getSolicitorReferences().getApplicantSolicitor1Reference();
+            return new HashMap<>(Map.of(
+                CLAIM_REFERENCE_NUMBER,
+                legacyCaseRef,
+                HEARING_DATE,
+                hearingDate,
+                HEARING_TIME,
+                hearingTime
+            ));
         }
-        return new HashMap<>(Map.of(
-            CLAIM_REFERENCE_NUMBER,
-            caseData.getLegacyCaseReference(),
-            HEARING_FEE,
-            caseData.getHearingFee() == null ? "£0.00" : String.valueOf(caseData.getHearingFee().formData()),
-            HEARING_DATE,
-            caseData.getHearingDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy")),
-            HEARING_TIME,
-            time.format(DateTimeFormatter.ofPattern("hh:mma")).replace("AM", "am").replace("PM", "pm"),
-            HEARING_DUE_DATE,
-            caseData.getHearingDueDate() == null ? "" :
-                caseData.getHearingDueDate().format(DateTimeFormatter.ofPattern("dd-MM-yyyy")),
-            CLAIMANT_REFERENCE_NUMBER, reference
+    }
+
+    private boolean isApplicantLip(CaseData caseData) {
+        return (YesOrNo.NO.equals(caseData.getApplicant1Represented()));
+    }
+
+    private String getRecipient(CaseData caseData, boolean isApplicantLip) {
+        return isApplicantLip ? caseData.getApplicant1().getPartyEmail()
+            : caseData.getApplicantSolicitor1UserDetails().getEmail();
+    }
 
-        ));
+    private String getReferenceTemplate(CaseData caseData, boolean isApplicantLip) {
+        return isApplicantLip ? String.format(REFERENCE_TEMPLATE_HEARING_LIP, caseData.getHearingReferenceNumber())
+            : String.format(REFERENCE_TEMPLATE_HEARING, caseData.getHearingReferenceNumber());
+    }
+
+    private String getEmailTemplate(CaseData caseData, boolean isApplicantLip) {
+        if (!isApplicantLip) {
+            if (caseData.getHearingFee() != null && caseData.getHearingFee().getCalculatedAmountInPence().compareTo(
+                BigDecimal.ZERO) > 0) {
+                return notificationsProperties.getHearingListedFeeClaimantLrTemplate();
+            } else {
+                return notificationsProperties.getHearingListedNoFeeClaimantLrTemplate();
+            }
+        } else {
+            return notificationsProperties.getHearingNotificationLipDefendantTemplate();
```

Comment:
Why are we getting the defendant template here?

Reply:
Because it is the same template for both and it is called "Defendant LIP Hearing Notice notification"